controllers/depresolver/depresolver.go

```diff
@@ -14,17 +13,58 @@ import (
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
-//Config holds operator configuration
+// EdgeDNSType specifies to which edge DNS is k8gb connecting
+type EdgeDNSType int
+
+const (
+	// DNSTypeNoEdgeDNS is default DNSType. Is used during integration testing when no infoblox or route53 exists.
+	// DNSTypeNoEdgeDNS is loaded when no Edge DNS configuration is provided
+	DNSTypeNoEdgeDNS EdgeDNSType = 1 << iota
+	// DNSTypeInfoblox type
+	DNSTypeInfoblox
+	// DNSTypeRoute53 type
+	DNSTypeRoute53
+)
+
+//Infoblox configuration
+//TODO: consider to make this private after refactor
+type Infoblox struct {
+	// Host
+	Host string
+	// Version
+	Version string
+	// Port
+	Port int
+	// Username
+	Username string
+	// Password
+	Password string
+}
+
+//Config is operator configuration returned by depResolver
 type Config struct {
-	//Reschedule of Reconcile loop to pickup external Gslb targets
+	// Reschedule of Reconcile loop to pickup external Gslb targets
 	ReconcileRequeueSeconds int
-	// Cluster Geo Tag to determine specific location
+	// ClusterGeoTag to determine specific location
 	ClusterGeoTag string
-	// Route53 switch
+	// ExtClustersGeoTags to identify clusters in other locations in format separated by comma. i.e.: "eu,uk,us"
+	ExtClustersGeoTags []string
+	// EdgeDNSType is READONLY and is set automatically by configuration
+	EdgeDNSType EdgeDNSType
+	// EdgeDNSServer
+	EdgeDNSServer string
+	// EdgeDNSZone main zone which would contain gslb zone to delegate; e.g. example.com
+	EdgeDNSZone string
+	// DNSZone controlled by gslb; e.g. cloud.example.com
+	DNSZone string
+	// DNSTypeRoute53 switch
+	//TODO: hide for depresolver subscriber as depresolver retrieves EdgeDNSType. Maybe we can change configuration and set EdgeDNSType directly instead of DNSTypeRoute53 boolean
```

Comment:
Let's make comments consistent (i.e `// TODO` instead of `//TODO`)

Reply:
THX Tim, I fixed TODOs. I use TODO now as points that will need to resolve in following PR in order to not touch live code during super large PR.

Reply:
@somaritane comments fixed